main.js

```diff
@@ -16,26 +16,47 @@ const removeClass = (el, oldClass) => {
   const inputField = document.getElementsByTagName('input')[0];
   let typeTimeout;
   const TYPE_INTERVAL = 10000;
+  const siteContainer = document.getElementsByTagName('body');
 
   inputField.addEventListener("input", (event) => {
     const value = event.target.value;
+    const prevFilter = document.getElementsByClassName('visible');
     if (typeTimeout) window.clearTimeout(typeTimeout);
+  
+    resetOldEmojis(prevFilter);
```

Comment:
@skimke So I think I've resolved the "emojis not related to the search" issue without editing `TYPE_INTERVAL`

Instead I moved the `resetOldEmojis` function to trigger anytime the input field is changed where before it was triggering if the input was empty and when we were trying to implement the search. This seems to have resolved the issue! Let me know if you're still seeing it.